app/presenters/flood_risk_engine/enrollment_presenter.rb

```diff
@@ -0,0 +1,45 @@
+#
+# A generalised presenter which adds helpers to simplify accessing enrollment data
+# from a view.
+#
+module FloodRiskEngine
+  class EnrollmentPresenter
+    delegate :exemption_location,
+             :correspondence_contact,
+             :organisation,
+             :org_type,
+             to: :enrollment
+    delegate :grid_reference,
+             to: :exemption_location, allow_nil: true
+    delegate :name,
+             :primary_address,
+             to: :organisation, allow_nil: true, prefix: true
+    delegate :email_address,
+             :telephone_number,
+             to: :correspondence_contact, prefix: true, allow_nil: true
+
+    def initialize(enrollment)
+      @enrollment = enrollment
+    end
+
+    def organisation_type
+      return unless org_type
+      I18n.t("organisation_types.#{org_type}")
+    end
+
+    def organisation_address
+      AddressPresenter.new(organisation_primary_address).to_single_line
+    end
+
+    def correspondence_contact_name
+      return unless correspondence_contact
+      title = correspondence_contact.full_name
+      title += " (#{correspondence_contact.position})" unless correspondence_contact.position.blank?
```

Comment:
I think `<<` is preferred to `+=` for string catenation.

http://stackoverflow.com/questions/4684446/why-is-the-shovel-operator-preferred-over-plus-equals-when-building-a